dynawo/sources/Common/DYNParameter.cpp

```diff
@@ -60,10 +60,8 @@ string origin2Str(const parameterOrigin_t& origin) {
       return "modelica file";
     case PAR:
       return "parameters";
-    case IIDM:
-      return "IIDM";
```

Comment:
We don't keep the IIDM information for referenceIIDM. Reading a bit the code, I get the impression that for referenceIIDM we create on the fly a parameter in the PAR API from Dynawo core (which seems strange for me).
So I believe we lose there the information on the origin, because then they should be considered as coming from the PAR file (?).

First, I really find it strange to do it this way - I understand we need to have the values read from IIDM to set up the parameters values and so it won't be possible to set the value in the API but perhaps we can have a structure without the value being set and only set the value later on. 
And I would say that it can be useful to know that the parameter comes from IIDM.

My comment is a bit messy because I don't really know what should be done but I don't really like the way we proceed for these referenceIIDM.

Reply:
Yes I also don't like it even though we can see in the par file that the parameter comes from  the iidm. We should keep the origin when iidm values are collected, but today it is not the case.